paper/myst.yml

```diff
@@ -15,6 +16,7 @@ project:
       email: doug@eos.ubc.ca
       orcid: 0000-0002-4327-2124
   doi: 10.1093/gji/ggad122
+  arxiv: 2211.14677
```

Comment:
@stevejpurves: is this the right place to put the arxiv link? I don't see it showing up in the preview